season_configs/spring_2017.yaml

```diff
@@ -167,7 +167,7 @@ info:
   anilist: 'https://anilist.co/anime/21860/ShuumatsuNaniShitemasukaIsogashiidesukaSukutteMoratteIidesuka'
   mal: 'https://myanimelist.net/anime/33502/Shuumatsu_Nani_Shitemasu_ka_Isogashii_desu_ka_Sukutte_Moratte_Ii_desu_ka'
 streams:
-  crunchyroll: ''
+  crunchyroll: 'http://www.crunchyroll.com/worldend-what-do-you-do-at-the-end-of-the-world-are-you-busy-will-you-save-us'
```

Comment:
@janclarin that's actually on Crunchy not me, most of their site (including all videos) doesn't support https.